### PR TITLE
Interface and Logic 

### PR DESCRIPTION
**Pull request by akshay-narkar**
## What?
Creating a Todolist app to create lists, add tasks and edit tasks using Javascript & Webpack
## Why?
Part of collaborative project at Javascript at Microverse
## How?
Mentioned in Readme 

## Changes Made Based on 1st Review
Removed Commented Code

### CODE REVIEW

File: src/modules/project.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+// Create project item/task
+const taskItem = (item) => ({
+  id: Date.now().toString(), item, date: dueDate.value, priority: priority.value, complete: false,
+});
+
+// Task form
+taskForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  e.target.style.display = 'none';
+  addTaskBtn.style.display = 'block';
```
Comment (rindrajosia) [inline src/modules/project.js]: Here for example you can create a CSS Class called .block and .none in your style.css then you can add to addTaskBtn

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- Good job by using ES6 syntax :+1: 
- Good job by using localStorage :100: 

To improve :factory_worker: [REQUIRED]:

- [x]  [REQUIRED] As your readme file is the first thing that users check, please make sure that your MIT link works.
- [x]  [REQUIRED] I got an error (Uncaught TypeError: f is null) when I create a task in the `Default project`. It will be nice if you fix it.


- [x]  [REQUIRED] Please, keep all the styling logic in CSS files.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)
- [x]  [REQUIRED] **- Task:** It has a title. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/todo_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list))
- **Interface:**  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/ui_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list))
- [x]  [REQUIRED] User can edit the details of a to-do item
- [x]  [REQUIRED] User can delete a to-do

Nice to have:

- [ ] 

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

File: dist/index.html [hunk 2]
```diff
@@ -1,98 +1,173 @@
...
+              </div>
+              <div class="form-group">
+                <input type="email" class="form-control" placeholder="Email Address" />
+              </div>
+              <button type="submit" class="btn btn-primary">Subscribe</button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div> -->
```
Comment (bellom) [inline dist/index.html]: Remove commented code;
You need to remove the commented code, It’s better to delete the commented code. If you need to review a previous version of your code, use git history.

File: dist/index.html [hunk 3]
```diff
@@ -1,98 +1,173 @@
...
   </body>
+
+  <script type="text/javascript" src="./main.js"></script>
+  <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
+  <!-- <script>
+    $(document).ready(function () {
+      $("#myModal").modal("show");
+    });
+  </script> -->
```
Comment (bellom) [inline dist/index.html]: Remove commented code;

File: src/dom.js [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+        content1.removeChild(content1.lastChild);
+      }
+    }
+  });
+
+  function deletelist(event) {
+    let remove = event.target.previousSibling.id;
+    remove = remove.slice(-1);
+
+    // const liststasks = JSON.parse(localStorage.liststore);
```
Comment (bellom) [inline src/dom.js]: Remove commented code

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team  :clap: 

**Great work making adding all the necessary features mentioned by the previous review.**

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔**

Project approved.
Great work.
You can merge and move on to other projects.👏👏👏

Happy coding💻💻💻

**Pull request merged**